docs/blog/newsletter-june-2021.rst

```diff
@@ -0,0 +1,137 @@
+
+-----------------------------------------------------------
+Priorities: Document new features or improve existing docs?
+-----------------------------------------------------------
+
+What would you prioritize: customer-requested improvements to existing documentation or working on new feature documentation? We discussed it on Slack this month as an interview question; for many documentarians though, the tension between priotizing new vs. existing docs is a day-to-day reality. Mostly, this means prioritizing which to work on *first* - because neither releasing new features without documentation nor neglecting to update existing documentation are good options.
+
+One reason to prioritize new features is if it's difficult/impossible to use undocumented features. In this month's discussion though, prioritizing improvements to existing docs had the most support.
+
+When you prioritize improvements to existing documentation, you raise the bar for new docs. The quality of your documentation rises as new docs are written to the higher standard. Making improvements implies figuring out what's wrong to begin with, so you'll be able to fix problems instead of repeating them. It's also easier to overlook or even forget low-quality content, whereas missing docs for new features are obvious. It makes sense to address existing customers' frustrations with existing docs. Even when poor docs do not represent a safety concern, they discourage current and potential customers. As one writer put it, "bad documentation is like breaking a promise".
+
+It's not just about keeping users happy, though, because your work has a business impact beyond providing the necessary documentation. For example, technical writers serve as a proxy for new users, so they can provide valuable insights from a user's perspective. The informal testing that writers often do during the documentation process can uncover bugs and usability issues to pass along to the product team. And good docs encourage self-service and reduce the number of customer support contacts. The benefits of good documentation spiral out to many other aspects of the business!
+
+-----------------------------------
+How to limit who can read your docs
+-----------------------------------
+
+A long-time community member started a new job recently, and among the novelties to explore they're figuring out how to allow only authorized users access to some of the docs.
+
+The community once again came to the rescue, with myriad options ranging from platform suggestions to details of how to roll your own or integrate different identity providers or workflows. They also cautioned wisely about making sure the solution was well-documented for future contributors to the docs!
+
+Here are all the options folks contributed to the discussion:
+
+* If you're writing docs in Confluence or knowledgebase articles in Zendesk, they support limited access
```

Comment:
Maybe "...both tools support..." instead of "...they support..." here?

Reply:
Fixed!